### PR TITLE
Simplifies the Circle request source

### PR DESCRIPTION
**Pull request by orta**
No tests yet, need my morning run

/cc @jamtur01

### CODE REVIEW

**Comment by orta** [comment]
Nice! They appeared at the exact same time, so we got double-danger'd.

File: lib/danger/ci_source/circle.rb [hunk 1]
```diff
@@ -32,53 +32,44 @@ def self.validates_as_pr?(env)
...
-
-    def fetch_pull_request_url(repo_slug, build_number)
-      build_json = client.fetch_build(repo_slug, build_number)
-      build_json[:pull_request_urls].first
-    end
-
     def initialize(env)
-      self.repo_url = GitRepo.new.origins # CircleCI doesn't provide a repo url env variable :/
+      # CircleCI doesn't provide a repo url env variable :/
+      self.repo_url = GitRepo.new.origins
```
Comment (jamtur01) [inline lib/danger/ci_source/circle.rb]: It does provide:

```env["CIRCLE_REPOSITORY_URL"]```

Which on my runs is:

```
CIRCLE_REPOSITORY_URL=https://github.com/kickstarter/kickstarter
```

**Comment by jamtur01** [comment]
Nice! Much simpler.

**Comment by orta** [comment]
I've created https://github.com/orta/show_circle_env which has examples of all env vars, in all of the types of API calls

**Comment by jamtur01** [comment]
+1.

**Pull request merged**